README.md

```diff
@@ -34,3 +34,7 @@ Las herramientas que se utilizarán para el desarrollo de este proyecto serán:<
 
 <img width="437" alt="Mockup user" src="https://github.com/user-attachments/assets/e85bebb7-9b75-448c-b4d0-74f734b3ee7b">
 <img width="437" alt="Mockup administration" src="https://github.com/user-attachments/assets/a8c85952-e9d2-464a-8c84-16c3343b5e88">
+
+## Proposed endpoint:
```

Comment:
It should in plural, as we are doing multiple endpoints

Reply:
the title has been changed